src/pages/Registration.js

```diff
@@ -61,14 +70,18 @@ const Registration = () => {
       <ModalBtn onClick={handleModal}>
         {moment(value).format('YYYY년 MM월 DD일')}
         <p>시간 넣을 자리</p>
-        <p>더보기</p>
+        <StyledIcon icon={faChevronDown} />
       </ModalBtn>
       {isOpen && (
         <DateBox>
-          <Calendar onChange={onChange} value={value} />
-          {TIME_DATAS.map(TIME_DATA => {
-            return <button key={TIME_DATA.id}>{TIME_DATA.time}</button>;
-          })}
+          <StyledCalendar onChange={onChange} value={value} />
+          <TagBtns>
+            {FILTERRING_BOX[0].select.map(TIME_DATA => {
+              return (
+                <button key={TIME_DATA.id}>{TIME_DATA.selectValue}</button>
+              );
+            })}
```

Comment:
```suggestion
            {FILTERRING_BOX[0].select.map(timeData=> {
              return (
                <button key={timeData.id}>{timeData.selectValue}</button>
              );
            })}
```
여기서 TIME_DATA를 상수데이터 네이밍 컨벤션으로 작성하신 이유가 있을까요?